soh/soh/Enhancements/enemyrandomizer.cpp

```diff
@@ -186,7 +186,7 @@ extern "C" uint8_t GetRandomizedEnemy(PlayState* play, int16_t *actorId, f32 *po
     // Lengthen timer in non-MQ Jabu Jabu bubble room.
     if (!isMQ && *actorId == ACTOR_OBJ_ROOMTIMER && *params == 30760 && play->sceneNum == SCENE_JABU_JABU &&
         play->roomCtx.curRoom.num == 12) {
-        *params = 92280;
+        *params = 26744;
```

Comment:
Seems a little odd given 30760 is above this, but looking at roomtimer code I'm pretty lost how this is all supposed to work at all

Reply:
From what I've discussed with Archez in the past, the mask of 0x3FF applied to params is what the timer actually becomes elsewhere. 30760 is just the base of the params without the timer applied. That being said, 92280 has never made sense to me, and I don't know what implications the change might have elsewhere.

Reply:
note 92280 here is being applied to `int16_t` so this line isn't changing anything

Reply:
Ah, I see, you brought it down to what it would technically overflow to with a -INT16_MAX, basically.

Reply:
Yea, I would like this value written as
```suggestion
        *params = (*params & ~0x3FF) | 120;
```

Preserves the non-timer part of the params and sets the timer part to 120 seconds, like it was doing before

Reply:
@Archez with that change can the hack be removed from ovl_Obj_Roomtimer init?

Reply:
> @Archez with that change can the hack be removed from ovl_Obj_Roomtimer init?

Indeed yes, that hack can be removed